testgen/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'testgen'
```

Comment:
you don't have to specify that usually, should work with the module name

Reply:
Looks like you're right, I even checked this couple times.
Previously gradle explicitely mentioned that I have to specify this file.
The only difference was that I ran it from inside the module.

Reply:
Actually no, here is what I receive:
```Project directory '/Users/bgaragulya/dev/openapi-boot-starter/testgen' is not part of the build defined by settings file '/Users/bgaragulya/dev/openapi-boot-starter/settings.gradle'. If this is an unrelated build, it must have its own settings file.
```

Reply:
strange, it works when I build with command line, and does not for IDE case